pkg/users/users.scss

```diff
@@ -133,6 +133,10 @@
     }
 }
 
+#accounts-create-dialog .pf-c-check {
+    margin-bottom: var(--pf-global--spacer--xs);
+}
+
```

Comment:
So I am not huge fan of this. I tried removing it and even experimented with using `body` property instead of `description`:
![Screenshot from 2022-03-10 12-34-57](https://user-images.githubusercontent.com/12330670/157654063-93ae2f13-f10a-4927-83f3-c161211eef97.png)

but the spacing on top is too much and on bottom too little :man_shrugging: 

I think we can just use utilities then - pushing changes